### ISSUE TITLE
Static Front Page won't change

### ISSUE THREAD

**Issue by Quigley-Brendan** [understrap/understrap#901]
Please delete anything that does not apply for you!

## Issue is about..
  - [ ] a Bug Report
  - [ ] a Feature Request
  - [ ] a Suggestion
  - [x] a Question
  - [X] I need help

## Please tell us about your environment:

  - **WordPress Version:** 5.0.3
  - **OS Version:** [MacOS [Version] | Android [Version] ]
  - **Browser:** [ all ]
---

# Issue Overview

I've taken on a WordPress site for a client, it's my first time working with Understrap. I tried to change the homepage to a different page, but the homepage still loads as the old page. 

I tried to find documentation to see if there is something I'm missing - theme settings etc. But I can't find anything on this. I doubt it's an actual bug. There are likely two reasons it isn't working for me. 

1. The theme has been modified in someway by the previous dev to always default the main page to a specific php template. I haven't found anything to suggest this - but I wouldn't recognize it quickly, so maybe I've overlooked something.

2. There is a unique process in understrap for changing the static front page. 

I'm asking to see if it's the second reason, and find out the right way to make the change.

**Comment by axlright** [comment]
>There is a unique process in understrap for changing the static front page.
There is not. I hope that helps.

**Comment by Quigley-Brendan** [comment]
For others in my situation. I was only looking at the child theme - not comparing it to the parent. Once I did I noticed that front-page.php was only in the child theme. 

According to this post https://wordpress.stackexchange.com/questions/110349/template-hierarchy-confused-with-index-php-front-page-php-home-php

That is a hardcoded front page. I renamed it to -old and now everything is working as expected.
